src/app/components/MediaLoader/index.test.tsx

```diff
@@ -168,4 +169,25 @@ describe('MediaLoader', () => {
       });
     });
   });
+
+  describe('Config', () => {
+    it('should use the counterNameOverride when rendering On Demand TV', async () => {
+      const buildConfigSpy = jest.spyOn(buildConfig, 'default');
+      await act(async () => {
+        render(
+          <MediaPlayer
+            blocks={onDemandTvBlocks as MediaBlock[]}
```

Comment:
Did we want to import the json from https://github.com/bbc/simorgh/blob/84335ed35db5ef6d240b426a6294a0aa3902c260/data/hindi/bbc_hindi_tv/tv_programmes/w13xttlw.json instead of duplicating the fixture data in src/app/components/MediaLoader/fixture.ts?

Reply:
As these are unit tests, we've sought to replicate the object created [here](https://github.com/bbc/simorgh/blob/1c1871222b1f47311d68123cfbc9d0ec50da07ab/src/app/routes/onDemandTV/getInitialData/index.js#L42) and passed down to the MediaLoader.
This is the rationale behind creating [this](https://github.com/bbc/simorgh/blob/1c1871222b1f47311d68123cfbc9d0ec50da07ab/src/app/components/MediaLoader/fixture.ts#L275) fixture.
On account of this, is it okay to retain `onDemandTvBlocks` in the test?

Reply:
Sure, it's very unlikely to change in the future, keeps our tests stable, and is consistent with the other tests in this file.